src/app/SharpRaven/Data/SentryEvent.cs

```diff
@@ -68,8 +70,15 @@ private SentryEvent()
         {
             Tags = new Dictionary<string, string>();
             Fingerprint = new List<string>();
+            this.contexts = Contexts.Create();
         }
 
+        /// <summary>Represents Sentry's structured Context</summary>
+        /// <seealso href="https://docs.sentry.io/clientdev/interfaces/contexts/"/>
+        public Contexts Contexts
+        {
+            get { return this.contexts; }
```

Comment:
Can't you just add an `internal set;` to this property so we get rid of the field?